CMakeLists.txt

```diff
@@ -513,9 +524,10 @@ SET(CPACK_SOURCE_STGZ "OFF")
 SET(CPACK_SOURCE_TBZ2 "ON")
 SET(CPACK_SOURCE_TGZ "OFF")
 SET(CPACK_SOURCE_TZ "OFF")
+SET(CPACK_PACKAGE_CONTACT "team@opencog.org")
```

Comment:
That mailing list doesn't exist (or is unused, unmaintained) Use <opencog@googlegroups.com> instead.